vars/unitTestPost.groovy

```diff
@@ -80,15 +86,9 @@ def call(Map config = [:]) {
   if (config['valgrind_stash']) {
     def valgrind_pattern = config.get('valgrind_pattern', '*.memcheck.xml')
     stash name: config['valgrind_stash'], includes: valgrind_pattern
-  } else {
-
-    // Need to leave this logic in here for backwards compatibility.
-    // Valgrind results need to stashed and reported in a common stage
-    // After all Valgrind tests are run.
-    valgrindReportPublish ignore_failure: ignore_failure,
-                          valgrind_stashes: []
   }
-  if (!stage_info['with_valgrind']) {
+  def no_record_issues = config.get('no_record_issues', true)
```

Comment:
The name of this variable seems to be inverted.  On line 93, `recordIssues` is being invoked when the conditional on line 91 `no_record_issues` is `true`, yes?  Shouldn't this variable (and the equivalent config option) be `record_issues`?

Reply:
Yes.  This is a compatibility issue, the code needs to work (for a time) with master daos but with this PR, if I change the case then legacy builds of master won't record the issues.

Reply:
But shouldn't the variable (and config argument) name be `record_issues` though?

Reply:
If it was record_issues then master daos with this PR would not have entered the conditional.

Reply:
I'm not talking about changing the logic in any way.  Just the variable/config item name.

The issue is that the name reads such that when `no_record_issue` == `true`, one believes that to mean that `recordIssues` will not be run.  However the logic on line 91 says that when `no_record_issue` == `true` `recordIssues` will be run.

Or am I totally missing something here?

Reply:
I did get quite confused when I was writing it.  I'll take another look at it tomorrow, it might just be that I can invert everything including the default and it would work.